src/app/views/homepage/homepage.component.ts

```diff
@@ -13,7 +13,7 @@ export class HomepageComponent {
 
   $=$;
   data =  homepageData;
-  slides=['../../../assets/images/Slide/slideshow1.jpg','../../../assets/images/Slide/slideshow2.jpg','../../../assets/images/Slide/slideshow5.jpg']
+  slides=['../../../assets/images/Slide/1.webp','../../../assets/images/Slide/2.webp','../../../assets/images/Slide/3.webp','../../../assets/images/Slide/4.webp','../../../assets/images/Slide/5.webp','../../../assets/images/Slide/6.webp','../../../assets/images/Slide/7.webp']
```

Comment:
Dont hardcode here. Declare and Import from homepage.ts like the newsJson and others